Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -382,7 +382,9 @@ def audit_deps
             problem "Dependency '#{dep.name}' is marked as :run. Remove :run; it is a no-op."
           end
 
-          next unless @new_formula && @official_tap
+          next unless @new_formula
+          next if formula.versioned_formula?
```

Comment:
Added this to match the `options_cop` check so it doesn't fail for deps and not options.